ably/options.go

```diff
@@ -443,6 +440,22 @@ func (opts *clientOptions) idempotentRESTPublishing() bool {
 	return opts.IdempotentRESTPublishing
 }
 
+func (opts *clientOptions) hasActiveInternetConnection() bool {
+	res, err := opts.httpclient().Get(internetCheckUrl)
+	if err != nil {
+		return false
+	}
+	data, err := ioutil.ReadAll(res.Body)
+	res.Body.Close()
+	if err != nil {
+		return false
+	}
+	if string(data) == internetCheckOk {
```

Comment:
This could just be `return string(data) == internetCheckOk`

Reply:
I will update